stock_valued_picking_report/views/stock_picking_view.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+
+    <record model="ir.ui.view" id="view_picking_form_valued">
+        <field name="name">Valued Stock Picking</field>
+        <field name="model">stock.picking</field>
+        <field name="inherit_id" ref="stock.view_picking_form"/>
+        <field name="arch" type="xml">
+            <field name="origin" position="after">
+                <field name="valued" invisible="1"/>
```

Comment:
Either you should remove attribute invisible=1 or you should change instructions in README file. With this attribute, user can't set valued field.

I'm in favour of removing attribute in order to allow user to print taxes directly in picking.